index.js

```diff
@@ -74,6 +77,12 @@ async function attemptPaymentToDestination({ destination, outChannelIds }) {
         outChannelIds
     }).catch(err => {
         console.error(err)
+        if (destination.PAYMENT_AMOUNT_SATS < 1) {
+            if ((paymentAmountSats / destination.PAYMENT_MULTIPLIER) > destination.MIN_PAYMENT_AMOUNT_SATS){
+                attempt++
+                await attemptPaymentToDestination({ destination, outChannelIds, attempt })
```

Comment:
Since this is recursive, this should be only one line:
```javascript
await attemptPaymentToDestination({ destination, outChannelIds, attempt + 1 })
```